components/availability/src/Availability.svelte

```diff
@@ -1456,6 +1456,13 @@
       },
     };
   }
+
+  //#region colours
+  // Show partial availability as a gradient, rather than as categorically "partial"
+  $: partialScale = scaleLinear()
+    .domain([0, allCalendars?.length / 2, allCalendars?.length])
+    .range([_this.busy_color, _this.partial_color, _this.free_color]);
```

Comment:
I don't think the yellow-greens look that great :( 

Mb merge now but share with Shashaank?